File: Add.java
import java.util.Scanner;

public class Add {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // Input two numbers
        System.out.print("Enter first number: ");
        int a = sc.nextInt();

        System.out.print("Enter second number: ");
        int b = sc.nextInt();

        // Calculate sum
        int sum = a + b;

        // Display result
        System.out.println("Sum of " + a + " and " + b + " is " + sum);

        sc.close();
    }
}

File: Add.c
#include <stdio.h>

int main() {
    int a, b, sum;

    // Input two numbers
    printf("Enter first number: ");
    scanf("%d", &a);

    printf("Enter second number: ");
    scanf("%d", &b);

    // Calculate sum
    sum = a + b;

    // Display result
    printf("Sum of %d and %d is %d\n", a, b, sum);

    return 0;
}
